amaranth/vendor/_gatemate.py

```diff
@@ -0,0 +1,129 @@
+from abc import abstractmethod
+from amaranth import *
+from amaranth.build import *
+from amaranth.lib.cdc import ResetSynchronizer
+
+
+
+__all__ = ["GateMatePlatform"]
+
+
+
+class GateMatePlatform(TemplatedPlatform):
+    """
+
+    Required tools:
+        * ``yosys``
+        * ``p_r``
+
+    The environment is populated by setting the ``AMARANTH_ENV_GATEMATE`` environment variable to point to the toolchain directory.
```

Comment:
This line is not accurate now either; you copied it before, modified it, and didn't fully revert the changes.

Reply:
I changed it to say 
```
    The environment is populated by running the script specified in the environment variable
    ``AMARANTH_ENV_GATEMATE``, if present.    
```
As I saw the format in the other vendor files.

Reply:
Thanks, this is OK.